internal/staging/util_test.go

```diff
@@ -20,17 +19,15 @@ var _ = Describe("Util", func() {
 				PipelineTriggerToken: "xyz",
 			}
 			MRiid = "87878"
-			gitlabClientGetter = nil
 		})
 
-		It("should not infer if false", func() {
-			gitlabConf.SetInferBranch(false)
+		It("interBranch=True defaultBranch=''", func() {
```

Comment:
Would it be better to add a description here like what will happen if users set this and that?